### PR TITLE
br: add s3 permission description

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add a description about br's s3 permission.
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs-cn/issues/6015

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -341,6 +341,8 @@ br restore full \
 
 如果需要恢复的数据并不是存储在本地，而是在 Amazon 的 S3 后端，那么需要在 `storage` 子命令中指定 S3 的存储路径，并且赋予 BR 节点和 TiKV 节点访问 Amazon S3 的权限。
 
+完成一次备份恢复需要的权限包括 `s3:ListBucket`, `s3:PutObject`，`s3:GetObject`。
```
Comment (overvenus) [inline br/use-br-command-line-tool.md:344]: Please update https://docs.pingcap.com/zh/tidb/stable/use-br-command-line-tool#%E5%A4%87%E4%BB%BD%E6%95%B0%E6%8D%AE%E5%88%B0-amazon-s3-%E5%90%8E%E7%AB%AF%E5%AD%98%E5%82%A8 too.

Also, could you add documentation of what permission TiKV needs?

Comment (3pointer) [reply br/use-br-command-line-tool.md:344]: This PR already update in the place that th link points to ... maybe your given link is wrong?

Comment (3pointer) [reply br/use-br-command-line-tool.md]: Done, PTAL again

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -164,6 +164,12 @@ br backup full \
 
 这里可以参照 [AWS 官方文档](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-bucket.html)在指定的 `Region` 区域中创建一个 S3 桶 `Bucket`，如果有需要，还可以参照 [AWS 官方文档](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-folder.html) 在 Bucket 中创建一个文件夹 `Folder`。
 
+> **注意：**
+>
+> 正常完成一次备份需要的最小权限包括 `s3:ListBucket`, `s3:PutObject`。
+>
+> 如果 TiKV 节点在备份期间遇到问题导致中断，则额外需要 `s3:AbortMultipartUpload` 权限进行中断操作。
```
Comment (overvenus) [inline br/use-br-command-line-tool.md:171]:
```suggestion
> **注意：**
>
> 正常完成一次备份 TiKV 和 BR 需要的最小权限包括 `s3:ListBucket`, `s3:PutObject` 和 `s3:AbortMultipartUpload`。
```

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -341,6 +347,10 @@ br restore full \
 
 如果需要恢复的数据并不是存储在本地，而是在 Amazon 的 S3 后端，那么需要在 `storage` 子命令中指定 S3 的存储路径，并且赋予 BR 节点和 TiKV 节点访问 Amazon S3 的权限。
 
+> **注意：**
+>
+> 完成一次恢复需要的最小权限包括 `s3:ListBucket`, `s3:GetObject`。
```
Comment (overvenus) [inline br/use-br-command-line-tool.md:352]:
```suggestion
> 完成一次恢复 TiKV 和 BR 需要的最小权限包括 `s3:ListBucket`, `s3:GetObject`。
```

**Review by overvenus — COMMENTED** [review]
rest LGTM

File: br/use-br-command-line-tool.md [hunk 4]
```diff
@@ -164,6 +164,10 @@ br backup full \
 
 这里可以参照 [AWS 官方文档](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-bucket.html)在指定的 `Region` 区域中创建一个 S3 桶 `Bucket`，如果有需要，还可以参照 [AWS 官方文档](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-folder.html) 在 Bucket 中创建一个文件夹 `Folder`。
 
+> **注意：**
+>
+> 正常完成一次备份 TiKV 和 BR 需要的最小权限包括 `s3:ListBucket`, `s3:PutObject` 和 `s3:AbortMultipartUpload`。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:169]:
```suggestion
> 要完成一次备份，通常 TiKV 和 BR 需要的最小权限包括 `s3:ListBucket`，`s3:PutObject` 和 `s3:AbortMultipartUpload`。
```

File: br/use-br-command-line-tool.md [hunk 5]
```diff
@@ -341,6 +345,10 @@ br restore full \
 
 如果需要恢复的数据并不是存储在本地，而是在 Amazon 的 S3 后端，那么需要在 `storage` 子命令中指定 S3 的存储路径，并且赋予 BR 节点和 TiKV 节点访问 Amazon S3 的权限。
 
+> **注意：**
+>
+> 完成一次恢复 TiKV 和 BR 需要的最小权限包括 `s3:ListBucket`, `s3:GetObject`。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:350]:
```suggestion
> 要完成一次备份，通常 TiKV 和 BR 需要的最小权限为 `s3:ListBucket` 和 `s3:GetObject`。
```

**Pull request merged**